_projects/2023/100517886/100517886.Rmd

```diff
@@ -0,0 +1,454 @@
+---
+title: "El crecimiento de la industria petrolera"
+description: Replication of a graph from El Orden Mundial which aims to represent the evolution of oil production since 1965. 
+categories: "2023"
+author: Sara Cristina Herranz Amado
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+git add
+git commit 
+git push
+en algun lugar tengo que añadir _project. 
+es lo que estuvimos haciendo el otro día
+
+
+Distill is a publication format for scientific and technical writing, native to the web.
+
+Learn more about using Distill at <https://rstudio.github.io/distill>.
+
+```{r}
+library(readr)
+library(readxl)
+library(tidyverse)
+library(ggplot2)
+library(ggstream)
+library(dplyr)
+library(ggtext)
+library(ggrepel)
+library(showtext)
+library(sysfonts)
+library(diagram)
+library(grid)
+library(plotrix)
+```
+
+## **CLEANING THE DATA**
+
+Uploading the data base, deleting unnecessary columns and NAs from the main variable (oilprod_kbd):
+
+```{r}
+data <-  read_xlsx(path = "Panel format.xlsx", sheet = 1, range = "A1:M7212")
+
+data <- data |>
+  select(-c(OPEC:CIS))
+
+panel <- data |> 
+  filter(!is.na(oilprod_kbd)) 
+  
+```
+
+First, we start by organizing the countries. We created a vector for the countries that needed to appear explicitly in our graph and then created a new database where all these countries appear. After that, we filtered the original database, but this time we selected the unwanted countries and summed them by region. That way, they would receive the name of the region.
+
+```{r}
+countries <- c("Australia", "China", "Indonesia", "Norway", "Russian Federation", "USSR", "United Kingdom", "Brazil", "Venezuela", "Mexico", "Canada", "US", "Saudi Arabia", "Iran", "Kuwait", "Iraq", "Libya", "United Arab Emirates", "Nigeria", "Other Africa", "Other Asia Pacific", "Other CIS", "Other Caribbean", "Other Eastern Africa", "Other Europe", "Other Middle Africa", "Other Middle East", "Other Northern Africa", "Other S. & Cent. America", "Other South America", "Other Western Africa")
```

Comment:
This line (and others) go out of the page. Please restyle your code to fit in the width of the column (see the resulting HTML. A maximum of 80 characters is generally advisable.